context/app/static/js/components/workspaces/WorkspacesList.jsx

```diff
@@ -50,7 +50,7 @@ function WorkspacesList() {
       <SpacedSectionButtonRow
         leftText={
           <Typography variant="subtitle1">
-            {workspacesList.length} Workspace{workspacesList.length && 's'}
+            {workspacesList.length} Workspace{workspacesList.length !== 1 && 's'}
```

Comment:
We still want it to read `Workspaces` if it's 0, right?

Reply:
Yeah, that seems right to me? But I don't care strongly.